### PR TITLE
Estelle/Arb Math Portfolio

### LINKED ISSUE #231: compute RMM-CC arbitrage size using `solstat`
To compute the optimal arbitrage between an RMM-CC Portfolio pool and the liquid exchange. This can be done analytically so there's a closed form formula.

- [ ] Write the function that computes the arbitrage size given a price difference.
- [ ] Call `solstat` and `solmate` contracts to do all the relevant math that we need to do to compute the arbitrage size.
- [ ] Write a test that has a `SimpleArbitrageur` look for price differences to compute this arbitrage amount for.

### PR DESCRIPTION
**Pull request by 0xEstelle**
Closes #231. This is still a work in progress but contains an abstraction of the pool parameterization so we have less hard coding instances, and currently stands at just the arb amount calculation for the x reserve. Working on the y reserve next.

### CODE REVIEW

**Comment by 0xEstelle** [comment]
lmao i forgot to put this as a draft. Not sure how to convert from here tbh

**Comment by Autoparallel** [comment]
Converted to draft for you.

**Comment by 0xJepsen** [comment]
How are things going on this front? Do you need any help? I see you cruising :)

**Review by Autoparallel — APPROVED** [review]
looks good, left a few comments

File: crates/cli/src/sim/portfolio/arbitrage.rs [hunk 1]
```diff
@@ -24,6 +29,171 @@ pub(crate) fn create_arbitrageur<S: Into<String>>(
...
+    ratio: U256,
+) -> Result<ComputeArbOutput, Box<dyn Error>> {
+    let manager = manager;
+    let admin = manager.agents.get("admin").unwrap();
+    let arbiter_math = manager.autodeployed_contracts.get("arbiter_math").unwrap();
+
+    let strike = U256::from(pool_params.strike);
+    let iv = U256::from(pool_params.volatility) * U256::from(10_u128.pow(18))
+        / U256::from(10u128.pow(4));
+    let tau = U256::from(31556953u128);
```
[the remark this answers is not in the document]
Comment (0xEstelle) [reply crates/cli/src/sim/portfolio/arbitrage.rs:58]: sure. seconds btw

File: crates/cli/src/sim/portfolio/arbitrage.rs [hunk 2]
```diff
@@ -24,6 +29,171 @@ pub(crate) fn create_arbitrageur<S: Into<String>>(
...
+    }
+}
+
+pub(crate) fn compute_arb_size(
+    manager: &mut SimulationManager,
+    pool_params: PoolParams,
+    delta_liquidity: i128,
+    pool_id: u64,
+    portfolio: &SimulationContract<IsDeployed>,
+    ratio: U256,
```
[the remark this answers is not in the document]
Comment (0xEstelle) [reply crates/cli/src/sim/portfolio/arbitrage.rs:49]: ratio of reference price to strike

**Pull request merged**